src/test/java/run/halo/app/controller/content/api/PostControllerTest.java

```diff
@@ -0,0 +1,66 @@
+package run.halo.app.controller.content.api;
+
+import static org.mockito.Mockito.when;
+
+import org.junit.jupiter.api.Assertions;
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.Test;
+import org.mockito.Mock;
+import org.mockito.MockitoAnnotations;
+import org.springframework.boot.test.context.SpringBootTest;
+import org.springframework.test.context.ActiveProfiles;
+import run.halo.app.model.dto.BaseCommentDTO;
+import run.halo.app.model.entity.PostComment;
+import run.halo.app.model.params.PostCommentParam;
+import run.halo.app.service.OptionService;
+import run.halo.app.service.PostCommentService;
+import run.halo.app.service.PostService;
+
+@SpringBootTest
+@ActiveProfiles("test")
+public class PostControllerTest {
+
+    private static final String COMMENT_CONTENT_TEST = "Hello world!";
+
+    @Mock
+    private PostService mockPostService;
+
+    @Mock
+    private PostCommentService mockPostCommentService;
+
+    @Mock
+    private OptionService mockOptionService;
+
+    @Mock
+    private BaseCommentDTO mockBaseCommentDTO;
+
+    @Mock
+    private PostComment mockPostComment;
+
+    private PostController postController;
```

Comment:
Meanwhile, we are not recommended that invoke controller method directly to test controller.

In this way, the parameters of API won't be checked really.

Please see also https://github.com/halo-dev/halo/blob/master/src/test/java/run/halo/app/it/BaseApiTest.java

Reply:
New references: https://spring.io/guides/gs/testing-web/

Reply:
Added corresponding integration tests for comment API.